_data/tutorial-groups.yml

```diff
@@ -0,0 +1,27 @@
+- id: networking
+  title: Networking
+  description: "Connecting ev3dev to a local network or the internet, and connecting to an ev3dev device from a PC."
+- id: system
+  title: "Ev3dev System"
+  description: "Configuring, updating and modifying an ev3dev installation."
+- id: website
+  title: This Website
+  description: "Using this website and contributing content."
+- id: hardware-motors
+  title: "Using Motors"
+  description: "Configuring and using motors on ev3dev, from the basic tacho motors to more complex third-party ones."
+- id: hardware-extras
```

Comment:
It depends. Does `hardware-extra` mean hardware that was not necessarily designed to be LEGO compatible? If so, the NxtMXX would be standard since it is designed for LEGO.